activitypub.php

```diff
@@ -90,7 +90,7 @@
 // Plugin related constants.
 \define( 'ACTIVITYPUB_PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
 \define( 'ACTIVITYPUB_PLUGIN_BASENAME', plugin_basename( __FILE__ ) );
-\define( 'ACTIVITYPUB_PLUGIN_FILE', plugin_dir_path( __FILE__ ) . '/' . basename( __FILE__ ) );
+\define( 'ACTIVITYPUB_PLUGIN_FILE', ACTIVITYPUB_PLUGIN_DIR . basename( __FILE__ ) );
```

Comment:
what about the `/` delimiter here?

Reply:
@obenland you seem to never make any issues, I should have known that it simply works as you changed it 🙄

Reply:
`plugin_dir_path()` [returns the path with a trailing slash](https://developer.wordpress.org/reference/functions/plugin_dir_path/#source).